#!/usr/bin/env python
PACKAGE = "robotic_arm"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("precise_positioning", bool_t, 0, "", False)
gen.add("acceleration_duration", int_t, 0, "", 200, 0, 5000)
gen.add("speed_multiplier", int_t, 0, "", 1, 1, 20)

exit(gen.generate(PACKAGE, "robotic_arm", "RoboticArm"))